client/driver/test-resources/qemu/README.md

```diff
@@ -0,0 +1,21 @@
+# QEMU Test Images
+
+## `linux-0.2.img`
+
+via https://en.wikibooks.org/wiki/QEMU/Images
+
+Does not support graceful shutdown.
```

Comment:
Is there a reason why we should keep both images around?

Reply:
The other one will live forever in git's history, so there's not much benefit in dropping it. It's also very small and fast, and so for tests that don't care about what's running in the VM it's ideal.